final release/format_data.py

```diff
@@ -0,0 +1,208 @@
+
+# coding: utf-8
+
+# In[45]:
+
+
+import numpy as np
+import pickle
+import matplotlib.pyplot as plt
+import h5py
+from skimage.transform import resize
+from astropy.io import ascii
```

Comment:
`ascii` is a Python builtin, so think of `ascii` as a reserved word. `import ascii as asc` or something like that.

Reply:
ok